---
title: "Expanding the Scope of Bacterial CRISPR Activation with PAM-Flexible dCas9 Variants"
collection: publications
permalink: /publication/9_Kiattisewee-Karanjia-2022
excerpt: 'This paper is part of Ice PhD research at UW.'
date: 2022-12-16
venue: 'ACS Synthetic Biology'
paperurl: ''
citation: '<u>Kiattisewee C.</u><sup>+</sup>, Karanjia A.V.<sup>+</sup>, Legut M., Daniloski Z., Koplik S.E., Nelson J., Kleinstiver B.P., Sanjana N.E., Carothers J.M.<sup>†</sup>, Zalatan J.G.<sup>†</sup> (2022). &quot;Expanding the Scope of Bacterial CRISPR Activation with PAM-Flexible dCas9 Variants.&quot; <i>ACS Synthetic Biology</i>. 11(12):4103-4112. PMID: 36378874.'
---

## Abstract

CRISPR-Cas transcriptional tools have been widely applied for programmable regulation of complex biological networks. In comparison to eukaryotic systems, bacterial CRISPR activation (CRISPRa) has stringent target site requirements for effective gene activation. While genes may not always have an NGG protospacer adjacent motif (PAM) at the appropriate position, PAM-flexible dCas9 variants can expand the range of targetable sites. Here we systematically evaluate a panel of PAM-flexible dCas9 variants for their ability to activate bacterial genes. We observe that dxCas9-NG provides a high dynamic range of gene activation for sites with NGN PAMs while dSpRY permits modest activity across almost any PAM. Similar trends were observed for heterologous and endogenous promoters. For all variants tested, improved PAM-flexibility comes with the trade-off that CRISPRi-mediated gene repression becomes less effective. Weaker CRISPR interference (CRISPRi) gene repression can be partially rescued by expressing multiple sgRNAs to target many sites in the gene of interest. Our work provides a framework to choose the most effective dCas9 variant for a given set of gene targets, which will further expand the utility of CRISPRa/i gene regulation in bacterial systems.

[Link to Publisher's site](https://pubs.acs.org/doi/10.1021/acssynbio.2c00405)
